workflow/scripts/sbs_process/max_filter.py

```diff
@@ -0,0 +1,15 @@
+from lib.sbs_process.max_filter import max_filter
+from skimage.io import imread, imsave
+
+# load log filtered image data
+log_filtered_data = imread(snakemake.input[0])
+
+# apply max filter
+max_filtered = max_filter(
+    log_filtered_data=log_filtered_data,
+    width=snakemake.params.width,
+    remove_index=snakemake.params.remove_index,
+)
+
+# Save the aligned data as a .tiff file
+imsave(snakemake.output[0], max_filtered)
```

Comment:
Are these working as you hoped? Did you check Fiji for compatibility?

Reply:
Yep, working as expected! They are compatible with Fiji, but let's start using napari as our default GUI to check images (as discussed over Slack)!